docs/limitations.md

```diff
@@ -1,14 +1,14 @@
-# Limitations in this version of SSE plugins
+# Limitations
 
-#### Returning data to Qlik
-There is no support for returning several columns or higher-order data, to Qlik than was sent to the plugin. The cardinality of the response from the plugin must be the same as sent from Qlik.
-- If less rows, or lower cardinality, is sent back from the plugin, Qlik will add null values to match the number of rows sent from Qlik to the plugin originally. Note that the mapping may not work as intended if less data is sent back.
-- If higher-order data is sent back, either too many rows or more than one column, Qlik will neglect the additional data and only take the first _n_ rows in the first column, assuming _n_ is the number of rows sent to the plugin. Note that the mapping may not work as intended if more data is sent back.
+#### Expressions using SSE must persist the cardinality
+If using SSE in a chart expression or in the Qlik load script (the `LOAD ... EXTENSION ...` statement excluded), you should preserve the cardinality and return a single column. In the case of aggregations, the response column should contain a single value (one row). In the case of tensor functions, the response column should contain the same number of rows as the request.
```

Comment:
When you use SSE in a chart expression or in the Qlik load script (the `LOAD ... EXTENSION ...` statement excluded), you must preserve the cardinality and return a single column. In the case of aggregations, the response column must contain a single value (one row). In the case of tensor functions, the response column must contain the same number of rows as the request.

Reply:
They actually do not have to, therefore we exchanged the word must to should, since it still works in some way even if they are doing it wrong. So I think we stick to the word should.